INMOOV-Head-Control/StandardServo.cpp

```diff
@@ -10,17 +10,44 @@ StandardServo::StandardServo(byte pin, float PWMPerDegree, int minRotation, int
   m_PWMPerDegree = PWMPerDegree;
```

Comment:
- What is the intention of using two constructor in the cpp file?
- Add default arguments to the constructor StandardServo::StandardServo() (just a reminder, ignore if you plan to add it later)